components/explore-the-data-page/DataDownloadButton.js

```diff
@@ -45,7 +45,20 @@ class DataDownloadButton extends React.Component {
         >
           Download (CSV)
         </A>
-        {downloadStarted && <SurveyModal />}
+        {downloadStarted && (
+          <Modal>
```

Comment:
What do you think about passing the content to these different modal elements as props? Here's some completely untested code as an example.

```
<Modal
  title={'Before you go...'}
  description={'TJI provides its data to all users for free. Please consider making a contribution so we can continue to be a free resource.'}
  action={() => (
    <button type="button" className="btn btn--primary" onClick={() => window.open('/donate', '_blank')}>
      Donate!
    </button>
  )}
/>
```

If we did that, we could move most of the modal styles into `styled-components` in `Modal` rather than including them globally.

Reply:
The reason I decided to use children rather than passing props was to keep the actual Modal component as simple and flexible as possible, so we just have a wrapper and can pass any content and layout we want into it.

I'm not opposed to passing it everything in as props though, but I just feel like we reduce the flexibility of it if we do.

Reply:
👍 I'm cool with keeping it flexible for now. If we implement another modal and find that the title, description, and action elements are things we'll implement in most modals, we can refactor to props later.

Reply:
Ah that's a great point about those being common elements for a modal.